sql/core/src/main/java/org/apache/spark/sql/sources/v2/DataSourceOptions.java

```diff
@@ -97,4 +130,20 @@ public double getDouble(String key, double defaultValue) {
     return keyLowerCasedMap.containsKey(lcaseKey) ?
       Double.parseDouble(keyLowerCasedMap.get(lcaseKey)) : defaultValue;
   }
+
+  public static final String KEY_PATH = "path";
+  public static final String KEY_TABLE = "table";
+  public static final String KEY_DATABASE = "database";
+
+  public Optional<String> getPath() {
```

Comment:
I think it is more friendly when using this in scala to drop the `get` and use just `path` or `database`.